s3/pkg/datastore/yig/conf/yig.toml

```diff
@@ -4,6 +4,7 @@ log_level=20
 
 [endpoint]
 url="default"
+machine_id=1
```

Comment:
Sorry, machine_id will be used for what, and why it is set to be 1?

Reply:
And why url is set to be 'default'? Does it need to be a traditional URL format string?

Reply:
@sfzeng machine_id is used for machine id section of snowflake algo. and the "default" url is used for testing.